src/components/ui/ProductTitle/ProductTitle.tsx

```diff
@@ -0,0 +1,38 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import './product-title.scss'
+
+interface ProductTitleProp {
+  /**
+   * A text such as the product's name.
+   */
+  title: ReactNode
+  /**
+   * A text to be used with a label such to be used with a label
+   */
+  label?: ReactNode
+  /**
+   * A text to be used below such as the product's reference number.
+   */
+  refNumber?: string
+}
+
+function ProductTitle({ title, label, refNumber }: ProductTitleProp) {
+  return (
+    <div className="product-title">
+      <div className="product-title__header">
+        {title}
+        {!!label && label}
+      </div>
+
+      {refNumber && (
+        <div className="product-title__footer / text-body-small">
```

Comment:
Are we having other elements here? If not, I think we can just call it `product-title__addendum`. What do you think?